web/pgadmin/settings/__init__.py

```diff
@@ -440,6 +441,31 @@ def get_tool_data(trans_id):
             ))
 
 
+@blueprint.route(
+    '/object_explorer_filter',
+    methods=["GET", "PUT"], endpoint='object_explorer_filter')
```

Comment:
Security Hotspot Error: "Allowing both safe and unsafe HTTP methods is security-sensitive"